mayara-core/src/protocol/navico.rs

```diff
@@ -350,8 +351,26 @@ pub struct Report02 {
     pub target_expansion: u8,   // 38
     _u06: [u8; 3],              // 39..42
     pub target_boost: u8,       // 42
-    _u07a: [u8; 32],            // 43..75 (split for serde array limit)
-    _u07b: [u8; 24],            // 75..99
+    _u07: [u8; 11],             // 43..54 unknown
+    // Guard zone fields (offsets 54-88)
+    pub guard_zone_sensitivity: u8, // 54 - shared by both zones (0-255)
+    pub guard_zone_1_enabled: u8,   // 55
+    pub guard_zone_2_enabled: u8,   // 56
+    _u08: [u8; 4],                  // 57..61 unknown (zeros)
+    pub guard_zone_1_inner_range: u8, // 61 - meters
```

Comment:
Doesn't it make more sense if the guard zone range is in meters in a u32 instead of a u8?